server.js

```diff
@@ -91,89 +121,76 @@ server.on("connection", function (socket) {
       player.tell("Dead man tells no tales")
       return
     }
+
+    if (isBannedCommand(getCommandAlias(command))) {
```

Comment:
Aren't paused commands using **keys** from `command_aliases`? If that is the case, then there should be another function, for translating `command` to command key and that should be passed to `isBannedCommand` instead of the command alias.

Reply:
Fixed, (`Server.getCommandKey`)